app/CoinDataStorageUtils.hs

```diff
@@ -115,6 +197,7 @@ pragma foreign_keys = ON; /*turn on foreign key constraints*/
 
 _initBody :: [Char]
 _initBody = "/* Database initialization Prologue */\n"
+  -- <> _dropTables
```

Comment:
Should we maybe make optional to _initBody receives a bool to dertermine if tables should be dropped?

Reply:
NVM, I see you already added _clearnAndInit method